src/main/java/org/folio/rest/impl/ModTenantAPI.java

```diff
@@ -99,8 +125,9 @@ private void handleModConfigurationGetResponse(HttpClientResponse response, Conf
       JsonObject jsonConfig = body.toJsonObject();
       JsonArray configs = jsonConfig.getJsonArray(CONFIGS);
       if (configs.isEmpty()) {
-        logger.info(String.format("Configuration group with configName %s isn't exist. " +
-          "Posting default configs for '%s' configuration group", MODULE_NAME, configName));
+        logger.info(String.format(
```

Comment:
Please use {} from log4j instead of String format

Reply:
> Please use {} from log4j instead of String format

Good note, thank you